benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm

```diff
@@ -0,0 +1,210 @@
+#### This is the unstable wedge experiment 2
+###  Experiment 2 tests how an unstable subcritical wedge deforms to reach the critical taper
+###  solution. Horizontal layers of sand have 10 cm shortening by inward movement of a mobile
+###  wall with a velocity of 2.5 cm/hour. Both the base and the surface are horizontal and the 
+###  “wedge” is thus in the unstable field. This experiment builds a thrust wedge through a 
+###  combination of mainly in-sequence forward and backward thrusting. Deformation starts by 
+###  forming a pop-up structure near the mobile wall.
+
+###  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400	#10 cm shortening
+set Use years in output instead of seconds = false
+set CFL number				   = 0.5
+set Maximum time step			   = 1e5 #3.6
```

Comment:
Why use a value 1e5, which will never be reached? I would just delete this line.